tests/func/test_data_cloud.py

```diff
@@ -290,6 +291,20 @@ def _get_cloud_class(self):
         return RemoteHDFS
 
 
+@pytest.mark.usefixtures("http_server")
+class TestRemoteHTTP(HTTP, TestDataCloudBase):
```

Comment:
It'd be so easy to also test with `TestDataCloudCLIBase` if we had splitted into smaller tests (`gc` will fail atm).

Reply:
@skshetry CLI tests are redundant here, we need to get rid of them at some point. Not worth creating it for http. `gc` fails because there is no `all` method implemented in http, it wasn't part of this ticket, unfortunately :slightly_frowning_face: